src/com/hp/ppm/integration/jira/JIRAIntegrationConnector.java

```diff
@@ -0,0 +1,60 @@
+package com.hp.ppm.integration.jira;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.hp.ppm.integration.FunctionIntegration;
+import com.hp.ppm.integration.IntegrationConnector;
+import com.hp.ppm.integration.IntegrationConnectorInstance;
+import com.hp.ppm.integration.ui.CheckBox;
+import com.hp.ppm.integration.ui.Field;
+import com.hp.ppm.integration.ui.LineBreaker;
+import com.hp.ppm.integration.ui.PasswordText;
+import com.hp.ppm.integration.ui.PlainText;
+
+public class JIRAIntegrationConnector implements IntegrationConnector {
```

Comment:
This is the most important class, so please add a javadoc comment at the class level that describes what this connector is, and what it can do. A few lines should be enough.